### PR TITLE
[Table] Add resizeRowsByTallestCell to handle multi-column resize

### LINKED ISSUE #1070: Apply resizeRowsByTallestCell to multiple columns
Currently resizeRowsByTallestCell operates on a column basis, so if i have a table

| col1 | col2 |
| ---- | ---- |
| text | text |
| {3 lines} | {2 lines} |

I want to resize so the second row is taller to fit all the contents. Today users would need to guess if they resize based on col1 or col2. Let's allow them to resize based on both and get the maximum across the columns 

@philcchen has generously suggested he contribute this

### PR DESCRIPTION
**Pull request by philcchen**
#### Checklist

- [x] Include tests

#### Changes proposed in this pull request:

Add a method called `resizeRowsByTallestCell` that takes in an array of column indices (or a single column index) to resize by one or more columns. If no array is provided, the method uses the columns in the viewport.

#### Screenshot

table-resize

### CODE REVIEW

**Comment by philcchen** [comment]
(testing the docs preview)

**Comment by philcchen** [comment]
@adidahiya yup, it was the fork; these previews seem to work

File: packages/table/src/table.tsx [hunk 1]
```diff
@@ -489,10 +489,26 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
...
+     */
+    public resizeRowsByTallestCell(columnIndices?: number | number[]) {
         const { locator } = this.state;
-
-        const tallest = locator.getTallestVisibleCellInColumn(columnIndex);
+        let tallest = 0;
+        if (columnIndices == null) {
+            // Consider all columns currently in viewport
+            const { grid } = this;
+            const { viewportRect } = this.state;
```
Comment (giladgray) [inline packages/table/src/table.tsx]: i prefer not destructuring variables that are used exactly once. can remove two lines by simply using `this.grid` once and `this.state.viewportRect` once (in the same line 😮 )

**Comment by cmslewis** [comment]
Updated the PR description to be consistent with the latest changes.

**Comment by giladgray** [comment]
description is incorrect--no method was added, but the signature of existing one changed to support more use cases.

**Pull request merged**